xml/monitoring.xml

```diff
@@ -111,6 +113,499 @@
+       </para>
+<screen>&prompt.user;wget http://&exampledomain;:9090/config --output-document=-</screen>
+      </step>
+     </stepalternatives>
+     <para>
+      Either of these methods should show the default contents of the
+      <filename>/etc/prometheus/prometheus.yml</filename> file.
+     </para>
+    </step>
+    <step>
+     <para>
+      Enable and start &grafana;:
+     </para>
+<screen>&prompt.root;systemctl enable --now grafana-server</screen>
+    </step>
+    <step>
+     <para>
+      Log in to the &grafana; web server at
+      <link xlink:href="http://&exampledomain;:3000"/>.
+     </para>
+     <para>
+      Use <literal>admin</literal> for both the user name and password, then
+      change your password when prompted.
+     </para>
+    </step>
+    <step>
+     <para>
+      On the left panel, select &graf-config-icon; and click <guimenu>Data Sources</guimenu>.
+     </para>
+    </step>
+    <step>
+     <para>
+      Click <guimenu>Add data source</guimenu>.
+     </para>
+    </step>
+    <step>
+     <para>
+      Find &prometheus; and click <guimenu>Select</guimenu>.
+     </para>
+    </step>
+    <step>
+     <para>
+      In the <guimenu>URL</guimenu> field, enter <literal>http://localhost:9090</literal>.
+      You can keep the default settings for the other fields.
+     </para>
+    </step>
+    <step>
+     <para>
+      Click <guimenu>Save &amp; Test</guimenu>.
+     </para>
+    </step>
+   </procedure>
+   <!-- Troubleshooting? What if Save & Test returns an error? -->
+   <para>
+    You can now configure &prometheus; to collect metrics from your cluster, and
+    add dashboards to &grafana; to visualize those metrics.
+   </para>
+  </sect2>
+  <sect2 xml:id="sec-monitoring-cluster-workloads">
+   <title>Monitoring cluster workloads</title>
+   <para>
+    You can monitor cluster workloads using the &prometheus; &slurm; exporter.
+    The &slurm; exporter gathers job and job queue status metrics from the
+    &slurm; workload manager. To visualize these metrics, you can import a
+    custom dashboard from the &grafana; website. For more information, see
+    <link xlink:href="https://grafana.com/grafana/dashboards/4323"/>.
+   </para>
+   <important>
+    <title>&slurm; exporter fails when GPU monitoring is enabled</title>
+    <para>
+     In &slurm; 20.11, the &slurm; exporter fails when GPU monitoring is enabled.
+    </para>
+    <para>
+     This feature is disabled by default. Do not enable it for this version of &slurm;.
+    </para>
+   </important>
+   <itemizedlist>
+    <title>Prerequisites</title>
+    <listitem>
+     <para>
+      <xref linkend="sec-installing-prometheus-grafana"/> is complete.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      The &slurm; workload manager is fully configured.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      You have internet access and policies that allow you to download the
+      dashboard from the &grafana; website.
```

Comment:
Do people usually not have web access in these scenarios? (I like it, but I just always assumed that SUSE docs usually would not mention web access as a prerequisite.)

Reply:
Sometimes customer sites have very restrictive access/firewalls/etc. While it is a super obvious requirement that I'd usually leave implied, in this case you're downloading something from a community site rather than an official SUSE repo, so it may run afoul of those restrictions.

Reply:
In the context of SES and SOC, we've specifically been told by Chinese customers that they often can't access the upstream Ceph and OpenStack docs normally due to the Great Firewall. I presume Chinese expect these kinds of thoroughly enjoyable shenanigans though.
But then, I guess in this case, even non-Chinese customers may run into this issue?

Reply:
Yep, for example, you might do offline installations and only your repo server has access to the internet. Or you might have very strict firewall restrictions because your servers handle sensitive data or military secrets or something.

Reply:
Ok, thanks for the explanations. -> setting resolved.